taxi/views.py

```diff
@@ -65,18 +67,18 @@
 class CarCreateView(LoginRequiredMixin, generic.CreateView):
     model = Car
     fields = "__all__"
-    success_url = reverse_lazy("taxi:car-list")
+    success_url = reverse_lazy("taxi:car_list")
```

Comment:
The URL name in `reverse_lazy` function should be written with `-` instead of `_`. Please, change `car_list` to `car-list`.